Description: Handle unexpected values for character passed to isalnum
Bug: https://github.com/hishamhm/htop/issues/711
Origin: upstream,https://github.com/hishamhm/htop/commit/03f17688ada3800bdd344ec4397f81289977b153
Author: Hisham Muhammad <hisham@gobolinux.org>
Last-Update: 2018-02-05

--- a/Panel.c
+++ b/Panel.c
@@ -469,7 +469,7 @@ HandlerResult Panel_selectByTyping(Panel* this, int ch) {
       this->eventHandlerState = xCalloc(100, sizeof(char));
    char* buffer = this->eventHandlerState;
 
-   if (ch < 255 && isalnum(ch)) {
+   if (ch > 0 && ch < 255 && isalnum(ch)) {
       int len = strlen(buffer);
       if (len < 99) {
          buffer[len] = ch;

